diff --git a/kivy/input/providers/hidinput.py b/kivy/input/providers/hidinput.py
index 926b2a3..8de7f0f 100644
--- a/kivy/input/providers/hidinput.py
+++ b/kivy/input/providers/hidinput.py
@@ -413,8 +413,8 @@ else:
             range_max_abs_y = 255
             range_min_abs_pressure = 0
             range_max_abs_pressure = 255
-            invert_x = int(bool(drs('invert_x', 0)))
-            invert_y = int(bool(drs('invert_y', 1)))
+            invert_x = int(bool(drs('invert_x', 1)))
+            invert_y = int(bool(drs('invert_y', 0)))
             rotation = drs('rotation', 0)
 
             def assign_coord(point, value, invert, coords):
